Library/Formula/gauche.rb

```diff
@@ -11,11 +11,23 @@ class Gauche < Formula
     sha1 "087b1f85a18485fe2226892d9b96066a027de606" => :lion
   end
 
+  option "with-slib", "Configure Gauche's slib module"
```

Comment:
Here it is https://github.com/Homebrew/homebrew/pull/35667

My concern is, a scheme implementation may depend on slib and slib also depends on the scheme implementation at the same time. How can we manage this kind of cyclic dependency?